test/trustedresources_test.go

```diff
@@ -106,6 +108,111 @@ func TestSignInterface(t *testing.T) {
 	}
 }
 
+func TestGetVerificationPolicy(t *testing.T) {
```

Comment:
This test feels pretty redundant to me; I personally think it could be removed. I made this comment because it seemed like we have functionality that handles warn mode policies, but we don't have test coverage for it. Can you add test coverage for that functionality instead?

Reply:
discussed offline, removed this